ci/aws-analytical-env/shared/meta.yml

```diff
@@ -214,3 +214,46 @@ meta:
       params:
         TF_CLI_ARGS_plan: -lock-timeout=300s
         TF_INPUT: "false"
+
+    vcs-to-s3:
+      task: vcs-to-s3
+      config:
+        platform: linux
+        image_resource:
+          type: docker-image
+          source:
+            repository: ((dataworks.docker_awscli_repository))
+            version: ((dataworks.docker_awscli_version))
+            tag: ((dataworks.docker_awscli_version))
+        params:
+          AWS_ROLE_ARN: arn:aws:iam::((aws_account.development)):role/ci
+          AWS_DEFAULT_REGION: ((dataworks.aws_region))
+        run:
+          path: sh
+          args:
+            - -exc
+            - |
+              export AWS_DEFAULT_REGION
+              source /assume-role
+              set -x
+              AWS_COMMOM_BUCKET="$(cat terraform-output-common/outputs.json |  jq -r '.config_bucket.value.id')"
+              cd chs-uc-lab-repo
```

Comment:
instead of this, it is better practice to use `dir: chs-uc-lab-repo` at the same level as `run:`

Reply:
Amended code to use `dir: chs-uc-lab-repo `